meow-helpers.el

```diff
@@ -226,5 +226,18 @@ This function produces several items:
 					    "meow--update-cursor")
 			    ,keymap))))
 
+(defun meow--mode-get-state ()
+  "Get initial state for current major mode."
+  (let ((modes (alist-get nil
```

Comment:
I think this first let binding is a bit hard to read. What do you think about this:

```
(cl-remove-if-not (lambda (mode)
                    (derived-mode-p (car mode)))
                  meow-mode-state-list)
```

Reply:
I removed the first binding entirely and just used meow-mode-state-list directly.